content/codespaces/getting-started-with-codespaces/getting-started-with-your-dotnet-project.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Getting started with your C# (.NET) project in Codespaces
+shortTitle: Getting started with your C# (.NET) project
+intro: 'You can create a custom dev container with all the tools necessary to get started with your C# (.NET) project in {% data variables.product.prodname_codespaces %}.'
+product: '{% data reusables.gated-features.codespaces %}'
+versions:
+  free-pro-team: '*'
+topics:
+  - Codespaces
+---
+
+
+### Introduction
+
+This guide shows you how to set up your C# (.NET) project in {% data variables.product.prodname_codespaces %}. It will take you through an example of opening your project in a codespace, and adding and modifying a dev container configuration from a template.
+
+#### Prerequisites 
+
+- You should have an existing C# (.NET) project in a repository on {% data variables.product.prodname_dotcom_the_website %}. If you don't have a project, you can try this tutorial with the following example: https://github.com/2percentsilk/dotnet-quickstart.
```

Comment:
Open to other sample repos - we want it to be relatively simple so I just used `dotnet new` to generate this application.